src/luascript.cpp

```diff
@@ -6726,16 +6726,30 @@ int LuaScriptInterface::luaCreatureGetMaster(lua_State* L)
 
 int LuaScriptInterface::luaCreatureSetMaster(lua_State* L)
 {
-	// creature:setMaster(master)
+	// creature:setMaster(master[, force = false])
 	Creature* creature = getUserdata<Creature>(L, 1);
 	if (!creature) {
 		lua_pushnil(L);
 		return 1;
 	}
 
 	Creature* master = getCreature(L, 2);
+	bool force = getBoolean(L, 3, false);
 	if (master) {
-		pushBoolean(L, creature->convinceCreature(master));
```

Comment:
It won't break it, the default for the force argument is false. It will work exactly the same when force is not set to true.